project-document/docs/database/database.md

```diff
@@ -122,6 +137,7 @@ Food is the object.
 | Access pattern name          | Key (PK/SK)             | How to Access      | Detail                                   | Context   |
 |:-----------------------------|:------------------------|:-------------------|:-----------------------------------------|:----------|
 | GetUser                      | UserId / User           | Get                | Get a single user data                   | User      |
+| GetUserByEMail               | EMailAddress            | Get                | Get a single user data                   | User      |
```

Comment:
I think this `How to access` should be `Query against GSI`. This GSI means `EMailUserRelation`.

Reply:
Thank you for your pointing out.
That's true.